CHANGELOG.md

```diff
@@ -198,6 +202,8 @@ This table maps the evolution of the billing data, showcasing column introductio
 | CommitmentDiscountName       | 1.0-preview             |                   |
 | CommitmentDiscountStatus     | 1.0                     |                   |
 | CommitmentDiscountType       | 1.0-preview             |                   |
+| ConsumedQuantity             | 1.0                     |                   |
+| ConsumedUnit                 | 1.0                     |                   |
 | ContractedCost               | 1.0                     |                   |
 | ContractedUnitPrice          | 1.0                     |                   |
 | EffectiveCost                | 0.5                     |                   |    
```

Comment:
What's the bar for listing an 'update' here? The vast majority (pretty much all) of the columns in the spec were updated in some way in v1.0. For example, half the columns changed from Column Required: True to Recommended in v1.0. Even outside of that change, many changed descriptions and some changed normative. I don't think this is a comprehensive list. 

I'm ok if we want to move this forward with it not being complete. I understand the tagged commit or branch may not have the complete changelog - but ultimately, unless someone is going through https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/candidate_recommendation...working_draft?diff=unified&w= and creating an accurate list, this feels like it's always going to be a 'best-effort' thing.

Reply:
I had the same debate. Should we say that, if a column is mentioned in the changelog, then it should be included in the summary?